.gitignore

```diff
@@ -1,4 +1,4 @@
-/node_modules/
+node_modules/
```

Comment:
I'm still not following.  Can you provide an example path, please?  To my understanding there shouldn't be any `node_modules` that aren't under the root `node_modules`